Packages/manifest.json

```diff
@@ -6,8 +6,7 @@
     "com.unity.collab-proxy": "2.0.4",
     "com.unity.feature.2d": "2.0.0",
     "com.unity.ide.rider": "3.0.21",
-    "com.unity.ide.visualstudio": "2.0.18",
-    "com.unity.ide.vscode": "1.2.5",
+    "com.unity.ide.visualstudio": "2.0.20",
```

Comment:
Looks like your computer config, may not be pushed ?

Reply:
This is actually because `com.unity.ide.vscode` is deprecated, and there is a newer patch version of the `com.unity.ide.visualstudio` package, which is the one to use now.